### PR TITLE
Wrong result type removed from ds-search command

### PR DESCRIPTION
**Pull request by rq-vsarode**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by rq-vsarode** [comment]
Not able to open form : https://forms.gle/XDfxU4E61ZwEESSMA

**Comment by rq-vsarode** [comment]
@yaakovpraisler can you please start review ?

**Comment by RotemAmit** [comment]
Hi @rq-vsarode,
As written in the above message ([link](https://github.com/demisto/content/pull/36754#issuecomment-2419442814)), the review will be slightly delay due to the holiday season in Israel.
Thank you for your patience.

File: Packs/DigitalShadows/ReleaseNotes/2_0_2.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### ReliaQuest GreyMatter DRP Incidents
+
+- ds-search issue fixed
```
Comment (yaakovpraisler) [inline Packs/DigitalShadows/ReleaseNotes/2_0_2.md:6]:
```suggestion
Fixed result type in ***ds-search*** command.
```

**Review by yaakovpraisler — APPROVED** [review]

**Comment by rq-vsarode** [comment]
Hi @yaakovpraisler can you please review again ? Also Please let me know what the next process

**Comment by yaakovpraisler** [comment]
Thank you @rq-vsarode, I'm merging this now and hope it will be in the marketplace by the end of the day.

**Comment by rq-vsarode** [comment]
Thanks

**Pull request merged**